### ISSUE TITLE
New scraper request

### ISSUE THREAD

**Issue by j6face** [Gujal00/GujalKodiWork#31]
Hi, Could you create new link for the following:

http://www.tamildhool.com/

http://www.tamilserialtoday.net/

http://www.tamiltwist.us/

Thanks. Great work!

**Comment by scmonline1** [comment]
May be in addition to above,

https://www.voot.com
&
https://www.sonyliv.com/

**Comment by j6face** [comment]
Hi, deccan delight 1.54 is not added in the repository 1.7
after i download and install it in kodi, it still shows deccan as 1.53

**Comment by Gujal00** [comment]
There are already over 700 installs of DD 1.5.4
Check your end, not an issue with the repository

**Comment by j6face** [comment]
I have downloaded the repository from this link and it should be directed to the 1.54, correct?
https://github.com/Gujal00/GujalKodiWork/releases/download/1.7/repository.gujal-1.7.zip

**Comment by Gujal00** [comment]
Yes, that is how everybody gets updates. Probably there is some caching server at your ISP with old data. try a force refresh or check for updates as relevant to the kodi version you are running.

**Comment by j6face** [comment]
thanks for the guide.

**Comment by scmonline1** [comment]
@Gujal00 ,
Gr8, thanks, will try it soon....

**Comment by scmonline1** [comment]
@Gujal00 
Thanks for the addon, working superb....

**Comment by Gujal00** [comment]
@j6face tamildhool added to DD 1.6.6

**Comment by Gujal00** [comment]
@scmonline1 
SonyLIV addon released. Install from my repository

**Comment by scmonline1** [comment]
Gr8,
Thanks for the efforts,
Will check ASAP,

Regards,
Sachin Mahajan

**Comment by Gujal00** [comment]
@j6face tamilserialtoday added to DD 1.6.7
